src/components/Swap/Settings/TransactionTtlInput.tsx

```diff
@@ -11,6 +11,7 @@ import { Label } from './components'
 
 const Input = styled(Row)`
   ${inputCss}
+  border: 1px solid ${({ theme }) => theme.outline};
```

Comment:
This should be backed out. Instead:
- Change the inputCss to use theme.outline for the border value. This will fix this issue, and proactively add a border around the token select search input (also spec'ed out in figma).
- In optionCss, on line 9, change theme.outline to `''` to avoid overwriting this, so that they stay in sync should inputCss change.